python/pythonmonkey/builtin_modules/base64.py

```diff
@@ -8,7 +8,10 @@
 
 
 def atob(b64):
-  return str(base64.standard_b64decode(b64), 'latin1')
+  # Workaround for pythons 'Incorrect padding' error when base64 decoding, see: 
+  # https://stackoverflow.com/questions/2941995/python-ignore-incorrect-padding-error-when-base64-decoding
+  append = b"==" if isinstance(b64, bytes) else "=="
```

Comment:
JS's `atob` function only accepts a string, so checking if it's a `bytes` type seems unnecessary.

Reply:
Fair point, mostly added this for sake of familiarity to typical python base64 usage, didn't want it to throw since you cannot concat strings with bytes and I'm not entirely sure how pythonmonkey's type coercion works internally, will change to only accept strings though.